src/errors.rs

```diff
@@ -33,28 +33,14 @@ pub enum InternalError {
     InternalInvariantFailed,
     #[error("Paillier error: `{0}`")]
     PaillierError(#[from] paillier::Error),
-    #[error("Failed to convert BigNumber to k256::Scalar, as BigNumber was not in [0,p)")]
-    CouldNotConvertToScalar,
-    #[error("Could not invert a Scalar")]
-    CouldNotInvertScalar,
     #[error("Reached the maximum allowed number of retries")]
     RetryFailed,
```

Comment:
This almost seems like it falls into a different category of error:  not enough resources or something similar? Though if the number of retries is hardcoded into the library, maybe this is just InternalInvariantFailed

Reply:
I feel like this is a blameless error, but the user is the one who has to do something about it.